README.md

```diff
@@ -35,3 +35,32 @@ Open the command palette and search for `Prettier: Auto format this file`
 ## Configuration
 
 The plugin takes the same settings and the `prettier` tool. See the [`prettier` repo](https://github.com/jlongster/prettier/blob/master/src/options.js) or [this repo](https://github.com/danreeves/sublime-prettier/blob/master/Prettier.sublime-settings). You can configure them in Sublime at `Preferences > Package Settings > Prettier`.
+
+You can turn off the auto formatting on save by setting `autoformat` to `false`.
+
+For example:
+
+```js
```

Comment:
Good point. I think it'll be fine if it still uses js highlighting but is JSON. e.g.:

```js
{
  // Comments are allowed in JS
  "printWidth": 80,
  // But not allowed in JSON
  "tabWidth": 2,
  "useFlowParser": false,
  "singleQuote": false,
  "trailingComma": false,
  "bracketSpacing": true
}
```

I _think_ Sublime handles comments anyway. Could be wrong though